### PR TITLE
[TASK] add initial documentation

### PR DESCRIPTION
**Pull request by kaystrobach**
related #83

### CODE REVIEW

File: Documentation/Developer/Index.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+================
+
+Build your own widget
+=====================
+
+.. todo::
+
+   define how developers can build their own widgets
+
+-------------------------
```
Comment (haassie) [inline Documentation/Developer/Index.rst]: This line is not needed. See https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/CheatSheet.html

File: Documentation/Developer/Index.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Extend predefined widgets
+-------------------------
+
+The extension ships a list of predefined widgets to ensure a consistent look of tables, graphs and data.
+
+The shipped widgets are in the folder `Classes/Widgets`.
+
+
+-------------------------------
+Make a completely custom widget
```
Comment (haassie) [inline Documentation/Developer/Index.rst]: To be honest, we don't want to promote this. We want people to use the predefined widget types as much as possible to make sure we keep an unified dashboard.

File: Documentation/Developer/Index.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+-------------------------
+Extend predefined widgets
+-------------------------
+
+The extension ships a list of predefined widgets to ensure a consistent look of tables, graphs and data.
+
+The shipped widgets are in the folder `Classes/Widgets`.
+
+
+-------------------------------
```
Comment (haassie) [inline Documentation/Developer/Index.rst]: This line is not needed

File: Documentation/Editor/Index.rst [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+===========
+For Editors
+===========
+
+Target group: **Editors**
+
+What is a widget?
+=================
+
+A widget consists of a canvas with a given size, which is enriched with data over time.
```
Comment (haassie) [inline Documentation/Editor/Index.rst]: I guess this is a bit too technical. Maybe just say something like:

A widget is an element on the dashboard that displays up to date information or gives you the possibility to easily execute a command.

File: Documentation/Editor/Index.rst [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+   Example Widget with TYPO3 News (version 1.0)
+
+What is a dashboard?
+====================
+
+A dashboard is a collection of configured widgets.
+Each user can have it's own dashboards.
+
+---------------------
```
Comment (haassie) [inline Documentation/Editor/Index.rst]: This can be removed

File: Documentation/Editor/Index.rst [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+Predefined Dashboards
+---------------------
+
+.. todo::
+
+   define how editors can interact with predefined dashboards
+
+-----------------------
```
Comment (haassie) [inline Documentation/Editor/Index.rst]: Same

File: Documentation/Index.rst [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+      en
+
+:Description:
+      Manual for our dashboard extension, which adds a flexible responsive dashboard.
+
+:Keywords:
+      dashboard, widgets
+
+:Copyright:
+      2018-2019
```
Comment (haassie) [inline Documentation/Index.rst]: Just 2019 in this case ;-)

File: Documentation/Installation/Index.rst [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+.. _installation:
+
+============
+Installation
+============
+
+Target group: **Administrators**
+
+This extension is part of the TYPO3 core.
```
Comment (haassie) [inline Documentation/Installation/Index.rst]: That's not the case in this manual. Please mention that this extension is a minimal version of the feature that will be released in core v10.

File: Documentation/Installation/Index.rst [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+
+.. code-block:: bash
+
+   friendsoftypo3/dashboard                                    1.0.0            Dashboard for TYPO3
+
+If it is not yet installed, use the ``composer require`` command to install the extension
+
+.. code-block:: bash
+
+   composer require friendsoftypo3/dashboard ^1.0
```
Comment (haassie) [inline Documentation/Installation/Index.rst]: I would leave out the version here as this is about to change quite often I guess.

File: Documentation/Installation/Index.rst [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+
+The given version depends on the version of the TYPO3 core you are using.
+
+Now head over to the extension manager and activate the extension.
+
+Installation without composer
+=============================
+
+In an installation without composer you may use the extension manager and search for the extension dashboard.
```
Comment (haassie) [inline Documentation/Installation/Index.rst]: This is not possible and we are thinking about not releasing it to TER at all to make people update to v10.

File: Documentation/Introduction/Index.rst [hunk 11]
```diff
@@ -0,0 +1,39 @@
+.. include:: ../Includes.txt
+
+
+.. _introduction:
+
+============
+Introduction
+============
+
+In this repository the TYPO3 Dashboard initiative will create a proof of concept of a Dashboard extension for TYPO3.
```
Comment (haassie) [inline Documentation/Introduction/Index.rst]: This is not a repository ;-)

File: Documentation/Meta/Index.rst [hunk 12]
```diff
@@ -0,0 +1,26 @@
+.. include:: ../Includes.txt
```
Comment (haassie) [inline Documentation/Meta/Index.rst]: I guess we don't need this file

File: Documentation/Settings.cfg [hunk 13]
```diff
@@ -0,0 +1,50 @@
+[general]
+
+; endless list of all of the general simple settings
+; you can use in 'conf.py'
+
+project     = Dashboard (dashboard)
+version     = latest
+release     = latest
+t3author    = TYPO3 Core Team + Contributors, TYPO3 Documentation Team
```
Comment (haassie) [inline Documentation/Settings.cfg]: For now this is: TYPO3 Dashboard Initiative + Contributors

File: Documentation/Developer/Index.rst (hunk 2 above)
Comment (kaystrobach) [reply Documentation/Developer/Index.rst]: so remove it

File: Documentation/Installation/Index.rst (hunk 8 above)
Comment (kaystrobach) [reply Documentation/Installation/Index.rst]: just removed it completely

File: Documentation/Meta/Index.rst (hunk 12 above)
Comment (kaystrobach) [reply Documentation/Meta/Index.rst]: was in the template, so would keep it or ask daniel from the docs team.

**Pull request merged**